dev-manual/env/compose.rst

```diff
@@ -195,6 +203,112 @@ stop and remove related containers, network and volumes by running:
 
    docker-compose down --volumes
 
+.. _docker-ldap-auth:
+
+LDAP Authentication
+===================
+
+Docker Configuration
+++++++++++++++++++++
+
+To enable LDAP authentication using docker, we need to create two new files and
+update the existing ``docker-compose.dev.yml`` file. These files are required to
+create a network for LDAP and to configure users for LDAP.
+
+First, create a custom network for authentication in a new file called
+``docker-compose.auth-network.yml`` and put it in the ``docker`` directory. Copy the
+following into the file:
+
+.. code-block:: bash
+
+   networks:
+      default:
+         name: auth-network
+         external: true
+
+Then set up docker compose to use the new network by appending the following to
+the end of the existing ``docker-compose.dev.yml`` file:
+
+.. code-block:: bash
+
+   networks:
+      auth-network:
+         name: auth-network
+
```

Comment:
I don't think this is necessary - Is this named network required to set this up?